.vscode/launch.json

```diff
@@ -26,11 +26,11 @@
       "type": "node",
       "request": "launch",
       "cwd": "${workspaceRoot}",
-      "program": "${workspaceRoot}/packages/salesforcedx-vscode-apex-debugger/out/src/adapter/apexDebug.js",
+      "program": "${workspaceRoot}/packages/salesforcedx-apex-debugger/out/src/adapter/apexDebug.js",
```

Comment:
Is there a naming convention that others have used to call their debug adapter NPM packages? Do they call it like apex-debug-adapter? If they do, maybe we can follow suit. 

I didn't check when I first created the name salesforce-apex-debugger.